src/components/Search/SearchResults/SearchResultItem.tsx

```diff
@@ -8,9 +8,12 @@ import styles from './SearchResultItem.module.scss';
 
 import Link from 'src/components/dls/Link/Link';
 import QuranWord from 'src/components/dls/QuranWord/QuranWord';
+import useGetChaptersData from 'src/hooks/useGetChaptersData';
+import { getChapterData } from 'src/utils/chapter';
 import { logButtonClick } from 'src/utils/eventLogger';
 import { toLocalizedVerseKey } from 'src/utils/locale';
 import { getChapterWithStartingVerseUrl } from 'src/utils/navigation';
+import { getChapterNumberFromKey } from 'src/utils/verse';
```

Comment:
Ditto: this is being shared with Tarteel Search results

Reply:
👍